docs/setup_guides/cloud.rst

```diff
@@ -100,45 +100,15 @@ Run the following from the commmand line
 
   gcloud auth login
 
-Now here is a bash script that will create a cluster corresponding to Pangeo
+A separate set of bash commands that will create a cluster corresponding to Pangeo
```

Comment:
Should there be some link word here, like "_Then, a separate set of bash commands will_"